app/src/main/java/com/kickstarter/viewmodels/CommentsViewModel.kt

```diff
@@ -184,20 +191,38 @@ interface CommentsViewModel {
 
             // TODO showcasing subscription to initialization to be completed on : https://kickstarter.atlassian.net/browse/NT-1951
             this.internalError
-                .compose(combineLatestPair(commentsList))
-                .filter { it.second.isEmpty() }
-                .map { it.first }
+                .compose(combineLatestPair(isFetchingData))
+                .filter {
+                    this.lastCommentCursor == null &&
+                        it.second == INITIAL_LOAD
+                }
+                .compose(bindToLifecycle())
                 .subscribe {
-                    it.localizedMessage
-                    Timber.d("************ On initializing error")
-                    this.initialError.onNext(it)
+                    this.initialError.onNext(it.first)
+                    this.isLoadingMoreItems.onNext(false)
                 }
 
-            // TODO showcasing pagination error subscriptionto be completed on : https://kickstarter.atlassian.net/browse/NT-2019
-            this.paginationError
+            // TODO showcasing pagination error subscription to be completed on : https://kickstarter.atlassian.net/browse/NT-2019
```

Comment:
you can remove the TODO as the work is already done:)

Reply:
done